vue-components/stories/Popover.stories.ts

```diff
@@ -34,10 +35,17 @@ export function basic( args ): Component {
 basic.args = {
 	isShown: false,
 	reactToHover: true,
+	position: 'bottom',
 	content: 'Here is some <em>content</em>. You can modify it in the "Controls" section.',
 };
 
 basic.argTypes = {
+	position: {
+		control: {
+			type: 'inline-radio',
+			options: PopoverPositions,
```

Comment:
Nitpick: I think if you make this `Object.values(PopoverPositions)` then the options might not be in all-caps in the controls section